mmv1/products/compute/RegionNetworkEndpointGroup.yaml

```diff
@@ -167,6 +167,8 @@ properties:
       - cloud_function
       - app_engine
       - serverless_deployment
+    allow_empty_object: true
+    send_empty_value: true
```

Comment:
```suggestion
```
I don't think we can specify empty block `cloud_run {}` since we require
```
at_least_one_of:
  - cloud_run.0.service
  - cloud_run.0.url_mask
```
https://github.com/Cheriit/magic-modules/blob/neg-app-engine/mmv1/products/compute/RegionNetworkEndpointGroup.yaml#L179